scripts/base/frameworks/dpd/__load__.zeek

```diff
@@ -1 +1,3 @@
-@load ./main
+@deprecated "Remove in v6.1 - use @load base/frameworks/analyzer instead";
```

Comment:
As this is now loaded in bare mode anyways, do want to just say "Remove in v6.1 - now loaded automatically"?

Reply:
And, yes, I'd keep this for now.